manifests/cf-manifest/manifest/800-logsearch.yml

```diff
@@ -64,11 +64,11 @@ jobs:
       - 10.0.17.14
   properties:
     redis:
-      host: ((queue_static_ips_second))
-    logstash:
-      output:
-        elasticsearch:
-          data_hosts: ((elasticsearch_master_static_ips))
+      host: ((queue_static_ips_first))
```

Comment:
Typo: This should be `queue_static_ips_second`.

This has resulted in both parsers pointed at one queue, and nothing pointed at the second queue.

Reply:
I'll raise a PR to fix

Reply:
ups, sorry :(